### PR TITLE
add addonplacementscore

### PR DESCRIPTION
**Pull request by haoqing0110**
The related feature request: open-cluster-management-io/community#52
Ref: https://github.com/open-cluster-management/backlog/issues/18210

Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

File: pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ var (
 		"cluster-manager/0000_01_clusters.open-cluster-management.io_managedclustersetbindings.crd.yaml",
 		"cluster-manager/0000_03_clusters.open-cluster-management.io_placements.crd.yaml",
 		"cluster-manager/0000_04_clusters.open-cluster-management.io_placementdecisions.crd.yaml",
+		"cluster-manager/0000_05_clusters.open-cluster-management.io_addonplacementscores.crd.yaml",
```
Comment (zhiweiyin318) [inline pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go:49]: there are ut and integration test cases about the count of resources deployed by the operator, need to update them too.

Comment (haoqing0110) [reply pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go:49]: @zhiweiyin318 thanks for the comment, all the testing cases are updated.

**Pull request merged**